src/services/tx/tx-sender/dispatch.ts

```diff
@@ -57,20 +61,47 @@ export const dispatchTxProposal = async ({
 /**
  * Sign a transaction
  */
+
+type SigningMethods = Parameters<Safe['signTransaction']>[1]
+
+/**
+ * It's not possible to determine if a hardware wallet is connected via MetaMask. We therefore
+ * try to sign with EIP-712 if a wallet supports it, otherwise falling back to `eth_sign`.
+ */
+const getSupportedSigningMethods = (wallet: ConnectedWallet): SigningMethods[] => {
```

Comment:
Here the Safe version should be taken into account as well.
`eth_sign` is only supported by Safes `>= v1.1.0`
https://github.com/safe-global/safe-react/blob/c413702a0ad5887b356f6bc6f30aabcfcba1882d/src/logic/safe/transactions/offchainSigner/index.ts#L21-L39

Reply:
Well, or not needed and when the loop is executed if Safe < v1.1.0 and eth_sign is tried this error will be returned: https://github.com/safe-global/safe-core-sdk/blob/40061dedc748525612cf30f2839ea194c8c3a1d2/packages/safe-core-sdk/src/Safe.ts#L474

Reply:
I've adjusted [`getSupportedSigningMethods`](https://github.com/safe-global/web-core/blob/iterate-signing-methods/src/hooks/wallets/wallets.ts#L105-L122) to take the Safe version into account. Let me know what you think.